install/kubernetes/cilium/charts/hubble-ui/templates/svc.yaml

```diff
@@ -0,0 +1,53 @@
+{{- if .Values.ui.enabled }}
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: hubble-grpc
+  namespace: {{ .Release.Namespace }}
+  labels:
+    k8s-app: hubble
+spec:
+  type: ClusterIP
+  clusterIP: None
+  selector:
+    k8s-app: hubble
+  ports:
+  - targetPort: 50051
+    protocol: TCP
+    port: 50051
+---
+kind: Service
+apiVersion: v1
+metadata:
+  namespace: {{ .Release.Namespace }}
+  name: hubble-ui
+spec:
+  selector:
+    k8s-app: hubble-ui
+  ports:
+    - name: http
+      port: 12000
+      targetPort: 12000
+  type: ClusterIP
+{{- end }}
+{{- if and .Values.metrics.enabled (.Values.metrics.serviceMonitor.enabled) }}
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: hubble-metrics
+  namespace: {{ .Release.Namespace }}
+  labels:
+    k8s-app: hubble-metrics
+spec:
+  clusterIP: None
+  type: ClusterIP
+  ports:
+  - name: metrics
+    port: {{ regexReplaceAll ":([0-9]+)$" .Values.metrics.address "${1}" }}
+    protocol: TCP
+    targetPort: metrics
+  selector:
+    k8s-app: hubble
```

Comment:
This also needs to target cilium-agent and does not belong to the `hubble-ui` chart. It should be part of the same chart as the `ServiceMonitor` object above.

Reply:
Moved it to cilium/agent/servicemonitor.yaml.

Reply:
Addressed review comments.

Reply:
This was actually not addressed, see: https://github.com/cilium/cilium/pull/10648#discussion_r408867449

Reply:
Fixed